Sources/BraveWallet/Crypto/Stores/NetworkStore.swift

```diff
@@ -39,6 +39,20 @@ public class NetworkStore: ObservableObject {
   private let rpcService: BraveWalletJsonRpcService
   private let walletService: BraveWalletBraveWalletService
   private let swapService: BraveWalletSwapService
+  
+  private weak var networkSelectionStore: NetworkSelectionStore?
+  
+  func openNetworkSelectionStore(mode: NetworkSelectionStore.Mode = .select) -> NetworkSelectionStore {
+    if let store = networkSelectionStore {
```

Comment:
You probably want to ensure that you give back a store that matches the mode passed in as well. If you can't have more than 1 open then maybe you can just close the current one and open a new one for the new mode?